src/i18n/translations/messages_en.json

```diff
@@ -137,5 +137,26 @@
   "Topics": "Topics",
   "Welcome to Zulip!\n\nYou'll need to first create an account from your computer. If you're not sure where to start, go to zulipchat.com from your web browser.\n\nHope to see you back here soon!\n": "Welcome to Zulip!\n\nYou'll need to first create an account from your computer. If you're not sure where to start, go to zulipchat.com from your web browser.\n\nHope to see you back here soon!\n",
   "Add subscribers": "Add subscribers",
-  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream."
+  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream.",
+  "English": "English",
```

Comment:
I am not sure we need to add the language names to be translated into all languages.
Do you think we should remove these?

Reply:
Sure. But then, maybe we don't need to have every language translated into every language? 
Even right now, this is 20 ^ 2 = 400 total translations. 
The idea of this work is that the user would recognize the language name in their own language.

Reply:
I see. If so, we may need to edit the requirements described in issue #2611. 

@gnprice your thoughts?